ctapipe/reco/sklearn.py

```diff
@@ -437,6 +436,9 @@ class ParticleClassifier(SKLearnClassificationReconstructor):
     property = ReconstructionProperty.PARTICLE_TYPE
 
     def __call__(self, event: ArrayEventContainer) -> None:
```

Comment:
subclasses should inherit docstrings from their parents automatically in sphinx I think. Was this needed?

Reply:
Oh yes, you are right. I was confused since it got overwritten everywhere else, but I think it would be better to just use the docstring from the parent class for `__call__` and `predict_table` in all subclasses here. 👍